src/components/Synthetics/ExecutionPriceRow.tsx

```diff
@@ -47,12 +47,12 @@ export function ExecutionPriceRow({
 
   const fullPositionPriceImpactBps =
     fees?.positionPriceImpact?.bps !== undefined && fees?.priceImpactDiff?.bps !== undefined
-      ? fees?.positionPriceImpact?.bps + fees?.priceImpactDiff?.bps * -1n
+      ? fees?.positionPriceImpact?.bps - fees?.priceImpactDiff?.bps
       : undefined;
 
   const positionPriceImpactDeltaUsd =
     fees?.positionPriceImpact?.deltaUsd !== undefined && fees?.priceImpactDiff?.deltaUsd !== undefined
-      ? fees?.positionPriceImpact?.deltaUsd + fees?.priceImpactDiff?.deltaUsd * -1n
+      ? fees?.positionPriceImpact?.deltaUsd - fees?.priceImpactDiff?.deltaUsd
```

Comment:
you still have unused `?`s
`fees.positionPriceImpact.deltaUsd - fees.priceImpactDiff.deltaUsd` should work

my dream is to get read of `?`s at all someday